.github/workflows/ci.yml

```diff
@@ -1,18 +1,17 @@
 name: Ambianic Functions API CI Test 
 
```

Comment:
@vickywane I don't see a few important steps in the CI:

- [ ] Tests against Netfliy function pre-deploy
- [ ] Semantic versioning, release and deployment of the Netfliy functions implementing the latest open api spec.
- [ ] semantic release that packages and published the client SDK to npm.
- [ ] semantic release of versioned docs, generated and published on github pages

Reply:
@vickywane you asked me to review this again, by you did not address some of my open questions. Please do that before you bring me back in. Thank you.

Reply:
@vickywane I thought you said all open comments are addressed. This one is still open...

Reply:
@vickywane which of the checkboxes in the list above have been closed and which ones are still open?
Simple check off the ones that are done so I can see what's ready for review.